node/lib/TranslatorInfo.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Information about a translator, as loaded from the translator's manifest.
+ */
+export class TranslatorInfo {
+    /**
+     * Fully-qualified name of the translator (required).
+     */
+    public readonly name: string;
+
+    /**
+     * List of interfaces implemented by the translator.
+     */
+    public readonly interfaceNames: string[];
```

Comment:
ah ok. Makes sense in JS/TS land :)

---
In reply to: [71909843](https://github.com/openT2T/opent2t/pull/1#discussion_r71909843) [](ancestors = 71909843,71904218)